.github/workflows/e2e-manual-icad.yaml

```diff
@@ -42,6 +43,7 @@ on:
           - v0.3.4
           - v0.2.3
           - v0.1.4
+          - v0.1.5
```

Comment:
This is the tag with the 1-minute packet timeout, right? Maybe we don't need to use it anymore, if we plan to test the timeout/reopen scenario only with simd >= 6.0.0.

Reply:
Since this is manual only, I don't think there's much harm in having this additional drop down option. I'm happy to have it removed though if people don't think it will ever be used!

Reply:
I would have to check, but I think v0.1.5 is basically the same as v0.1.4 but with a shorter timeout, so I think it doesn't really add much to have both...

Reply:
okay sounds good to me to remove it 👍